docs/terminology/general-terminology.md

```diff
@@ -45,29 +40,35 @@ The base peak chromatogram (BPC) displays the signal intensity of the most inten
 The extracted ion chromatogram (EIC) displays the signal intensity of a specific m/z value, within a defined tolerance (e.g. ± 5 ppm), at any one retention time point in the LC-MS run. The following figure shows the EIC of _m/z_ 455.2945 ± 5 ppm (same sample as above).
 ![EIC](EIC.png)
 
+### Chromatographic deconvolution
+Peak overlapping, or co-elution, is a common problem in any chromatographic separation technique. In the case of LC-MS (especially untargeted _omics_ analysis), it is virtually impossible to obtain a full baseline separation for the hundreds (or thousands) of analytes eluted through the column. The process of XXX is called _chromatographic deconvolution_ and is one of the most crucial steps of data processing. TO FINISH
+
 # MZmine-specific terminology
 
 ### Masses and Features
-Mass is ...individual signal in a mass spectrum, which corresponds to an ion detected by the mass spectrometer.
-
-In LC-MS, a 'feature' is defined as a pair of m/z and retention time, normally associated with a signal intensity.
+In MS data processing, the term _mass_ is normally used to refer to an individual signal in a mass spectrum, which corresponds to an ion detected by the mass spectrometer (see [Mass detection](../module_docs/featdet_mass_detection/mass-detection.md)).
 
-The term ‘feature’ is used to emphasize the 3D nature of the signal, as opposed to the term ‘peak’, which is typically used for 2D datasets (e.g., m/z in a mass spectrum).
+In LC-MS, a _feature_ is defined as a bounded, two-dimensional (_m/z_ and RT dimensions) signal characterized by a pair of _m/z_ and RT values and associated with the detected signal intensity. In the case of LC-IM-MS data, a feature is also characterized by the ion mobility value recorded for the ion (see [LC-MS and LC-IMS-MS data comparison](../workflows/imsworkflow/lc-ms-and-lc-ims-ms-data-comparison.md). MZmine 3 provides a selection of different algorithms for LC-(IM)-MS feature detection, depending on the nature of the MS data (_e.g._ mass accuracy and resolution). However, they all follow the same logic: [EICs](#extracted-ion-chromatogram) are constructed starting from each _m/z_ value in the mass lists and subsequently deconvoluted into individual features (see figure).
```

Comment:
deconvoluted -> resolved
Add:
Further, additional information, such as isotope patterns can be assigned to a feature.